pkg/machines/components/storagePools/createStoragePoolDialog.jsx

```diff
@@ -229,6 +237,44 @@ const StoragePoolSourceRow = ({ onValueChanged, dialogValues }) => {
                 <hr />
             </React.Fragment>
         );
+    else if (dialogValues.type == 'disk')
+        return (
+            <React.Fragment>
+                <label className='control-label' htmlFor='storage-pool-dialog-source'>
+                    {_("Source Path")}
+                </label>
+                <FormGroup
```

Comment:
This really is nit-picky but the layout of `<Tag attribute=...` differes in 3 different ways across this fragment (One starting with a linebreak, one where the attributes are indented relative to the tagname, and one where attributes are indented relative the the first attribute).

Personally I prefer 
```
<Tag attribute1=
     attribute2=
     ...
```

If you'd prefer not to bother that's also fine :D

Reply:
I will fix it, thanks :D